### DISCUSSION TITLE
Is there a way to exit process-compose if a process is skipped?

### DISCUSSION THREAD

**Discussion by shivaraj-bh** [F1bonacc1/process-compose#221] [Q&A] [answered]
If there is not already a way, I’d be happy to raise a PR.

**Comment by F1bonacc1** [comment, marked as answer]
Hi @shivaraj-bh,

Today it is not possible, but I am considering 2 possible implementation approaches:
1. Add something like `exit_on_skipped` flag that will allow to do just that.
2. To add another type of dependency between processes: `process_skipped`, which will allow you to do that and additional cleanup actions if something goes wrong.

The first one should be very straightforward.
The second might be a bit complicated to implement and might be a very low ROI if it's a very narrow use case.

**Reply by shivaraj-bh** [reply]
I am going with 1 for now, since that seems simpler. I will create an issue from this discussion and raise a PR to resolve it.
